lib/Controller/ConfigController.php

```diff
@@ -16,26 +17,33 @@
 use OCP\IRequest;
 use OCP\AppFramework\Http\DataResponse;
 use OCP\AppFramework\Controller;
+use Psr\Log\LoggerInterface;
 
 use OCA\Collaboard\Service\CollaboardAPIService;
 use OCA\Collaboard\AppInfo\Application;
 use OCP\PreConditionNotMetException;
 
-class ConfigController extends Controller {
+class ConfigController extends Controller
+{
```

Comment:
Nitpicking: Let's stick with inline opening block character everywhere.
```suggestion
class ConfigController extends Controller {
```

Reply:
Argh.. That the default formatting option for the php plugin I'm using with VSC. I'll set up composer to get cs:fix going...